modules/javafx.graphics/src/main/java/javafx/css/TransitionEvent.java

```diff
@@ -100,10 +100,26 @@ public final class TransitionEvent extends Event {
      *
      * @param eventType the event type
      * @param property the {@code StyleableProperty} that is targeted by the transition
-     * @param propertyName the name of the targeted CSS property or sub-property
      * @param elapsedTime the time that has elapsed since the transition has entered its active period
      * @throws NullPointerException if {@code eventType}, {@code property} or {@code elapsedTime} is {@code null}
      */
+    @Deprecated(forRemoval = true)
```

Comment:
Can you add `since = 24,` before the `forRemoval`?

Also please add an `@deprecated` javadoc tag referring to the new constructor.